WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -225,9 +226,9 @@ async Task<(AliceClient? AliceClient, PersonCircuit? PersonCircuit)> RegisterInp
 		}
 
 		// Gets the list of scheduled dates/time in the remaining available time frame when each alice has to be registered.
-		var remainingTimeForRegistration = (roundState.InputRegistrationEnd - DateTimeOffset.UtcNow);
+		var remainingTimeForRegistration = roundState.InputRegistrationEnd - DateTimeOffset.UtcNow;
 
-		Logger.LogDebug($"Round ({roundState.Id}): Input registration started, it will end in {remainingTimeForRegistration:hh\\:mm\\:ss}.");
+		Logger.LogDebug($"Round ({roundState.Id}): Input registration started - it will end in: {remainingTimeForRegistration:hh\\:mm\\:ss}.");
```

Comment:
I was thinking maybe of an improvement - to see (besides the remaining time?) the exact time when will it be finished.

To add something like `DateTime.Now + remainingTimeForRegistration`?

Reply:
@molnard WDYT?

Reply:
In the long term, this message will be removed, it is only for debugging purposes. For me, the current message is enough. @Szpoti if you debug a lot and that is really missing for you, both can be displayed. Is that the case?